### PR TITLE
feat: add new icon mdiClipboardAlert

### PR DESCRIPTION
**Pull request by lpatel-eightfold**
## SUMMARY:
Adding new mdi icon mdiClipboardAlert

## GITHUB ISSUE (Open Source Contributors)
NA

## JIRA TASK (Eightfold Employees Only):
https://eightfoldai.atlassian.net/browse/ENG-48809

## CHANGE TYPE:

- [ ] Bugfix Pull Request
- [X] Feature Pull Request

## TEST COVERAGE:

- [X] Tests for this change already exist
- [ ] I have added unittests for this change

## TEST PLAN:
- Tested in PR codesandbox
Code:
```js
import "./styles.css";
import { IconName, Icon, IconSize } from "@eightfold.ai/octuple";

export default function App() {
  return (
    <div className="App">
      <Icon path={IconName.mdiClipboardAlert} color="green" />
    </div>
  );
}
```

### CODE REVIEW

File: src/components/Icon/mdi.ts [hunk 1]
```diff
@@ -24,10 +24,11 @@ export enum IconName {
...
+  mdiAlarm = 'M12,20A7,7 0 0,1 5,13A7,7 0 0,1 12,6A7,7 0 0,1 19,13A7,7 0 0,1 12,20M12,4A9,9 0 0,0 3,13A9,9 0 0,0 12,22A9,9 0 0,0 21,13A9,9 0 0,0 12,4M12.5,8H11V14L15.75,16.85L16.5,15.62L12.5,13.25V8M7.88,3.39L6.6,1.86L2,5.71L3.29,7.24L7.88,3.39M22,5.72L17.4,1.86L16.11,3.39L20.71,7.25L22,5.72Z',
   mdiAlert = 'M13 14H11V9H13M13 18H11V16H13M1 21H23L12 2L1 21Z',
   mdiAlertCircle = 'M13,13H11V7H13M13,17H11V15H13M12,2A10,10 0 0,0 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2Z',
   mdiAlertCircleOutline = 'M11,15H13V17H11V15M11,7H13V13H11V7M12,2C6.47,2 2,6.5 2,12A10,10 0 0,0 12,22A10,10 0 0,0 22,12A10,10 0 0,0 12,2M12,20A8,8 0 0,1 4,12A8,8 0 0,1 12,4A8,8 0 0,1 20,12A8,8 0 0,1 12,20Z',
+  mdiAlertOctagram = 'M2.2,16.06L3.88,12L2.2,7.94L6.26,6.26L7.94,2.2L12,3.88L16.06,2.2L17.74,6.26L21.8,7.94L20.12,12L21.8,16.06L17.74,17.74L16.06,21.8L12,20.12L7.94,21.8L6.26,17.74L2.2,16.06M13,17V15H11V17H13M13,13V7H11V13H13Z',
```
Comment (jhoward-eightfold) [inline src/components/Icon/mdi.ts:31]: is this the icon?
https://icon-sets.iconify.design/mdi/alert-octagram/

Why not use one of the more normal alarm icons already included?

Comment (lpatel-eightfold) [reply src/components/Icon/mdi.ts:31]: yes, we didn't use the circle one to keep it distinct from all other icons we are using.
These are the icons we need to use: [figma](https://www.figma.com/file/tmD6ugMiv2rBLxCWSe7xeD/Display-Failure--Flex?node-id=1%3A2&t=UASNrdSefIAKWtQy-1)

Comment (dkilgore-eightfold) [reply src/components/Icon/mdi.ts:31]: Let's hold-off until we know.

**Review by dkilgore-eightfold — CHANGES_REQUESTED** [review]

File: src/components/Icon/mdi.ts (hunk 1 above)
Comment (lpatel-eightfold) [reply src/components/Icon/mdi.ts:31]: We will use the X in circle icon, removing this icon

**Review by jhoward-eightfold — APPROVED** [review]

**Review by dkilgore-eightfold — APPROVED** [review]

**Pull request merged**